src/test/java/com/acrolinx/sidebar/utils/MatchUtilsTest.java

```diff
@@ -70,20 +70,23 @@ void shouldNotRemoveWhiteSpacesIfRepresentedCorrectlyByOffsets() {
         MatchUtils.filterInsignificantWhitespaceMatches(acrolinxMatches, lastCheckedContent));
   }
 
-  private void verifyMatches(
-      List<? extends AbstractMatch> expected, List<? extends AbstractMatch> actual) {
-    assertEquals(expected.size(), actual.size());
-    for (int i = 0; i < expected.size(); i++) {
-      final AbstractMatch expectedMatch = expected.get(i);
-      final AbstractMatch actualMatch = actual.get(i);
+  private static void verifyMatches(
+      List<? extends AbstractMatch> expectedAbstractMatches,
+      List<? extends AbstractMatch> actualAbstractMatches) {
+    assertEquals(expectedAbstractMatches.size(), actualAbstractMatches.size());
+    for (int i = 0; i < expectedAbstractMatches.size(); i++) {
+      verifyMatch(expectedAbstractMatches.get(i), actualAbstractMatches.get(i));
+    }
+  }
 
-      assertTrue(expectedMatch.getContent().equals(actualMatch.getContent()));
+  private static void verifyMatch(
+      AbstractMatch expectedAbstractMatch, AbstractMatch actualAbstractMatch) {
+    assertTrue(expectedAbstractMatch.getContent().equals(actualAbstractMatch.getContent()));
```

Comment:
use `assertEquals()` instead of `assertTrue()`

Reply:
Fixed